Core/src/IfcGeometryConverter/ProfileConverter.h

```diff
@@ -1652,6 +1655,7 @@ namespace OpenInfraPlatform {
 		protected:
 
 			std::shared_ptr<PlacementConverterT<IfcEntityTypesT>> placementConverter;
+			std::shared_ptr<SplineConverterT<IfcEntityTypesT>> splineConverter;

```

Comment:
If CurveConverter gets splineconverter removed as member, these can be removed as well.

Reply:
Done in commit b13e888